libs/content/workspace/src/show.rs

```diff
@@ -793,7 +772,7 @@ impl Workspace {
                     .expand(2.0);
             let close_button_resp = ui.interact(
                 close_button_rect,
-                Id::new("tab label close button").with(self.tabs[t].id),
```

Comment:
fix: distinguish close buttons by tab index instead of file id to fix close button for mind map